modules/460-log-shipper/crds/cluster-log-destination.yaml

```
@@ -396,6 +404,62 @@ spec:
                           type: boolean
                           default: true
                           description: Validate the TLS certificate of the remote host.
+                splunk:
+                  type: object
+                  required:
+                  - token
+                  - endpoint
+                  properties:
+                    endpoint:
+                      type: string
+                      description: The base URL of the Splunk instance.
```

```suggestion
                      description: Base URL of the Splunk instance.
```
This one is removed for consistency, given that articles are omitted in all similar phrases below.

---

Will be applied in a separate PR.